R/Connect.R

```diff
@@ -557,7 +593,9 @@ connectUsingJdbcDriver <- function(jdbcDriver,
                     stringQuote = stringQuote,
                     dbms = dbms,
                     uuid = uuid)
-  registerWithRStudio(connection)
+  if (dbms != "spark") {
+    registerWithRStudio(connection)  
```

Comment:
Why not register the connection with RStudio?

Reply:
It errors when trying against our Databricks cluster. I thought for now, I'd do this hack till I can figure out a solution

Reply:
Would you mind fixing the error instead of having the workaround? I can already see this being overlooked if we try to merge the spark branch into develop

Reply:
Ah, looks like the fix was easy, just needed to include spark in the hasCatalogs function.